packages/devextreme-angular/src/common/charts/index.ts

```diff
@@ -0,0 +1,46 @@
+export {
```

Comment:
Why is it duplicated instead of moving from https://github.com/GoodDayForSurf/DevExtreme/blob/24_1_angular_regenerate_for_ng18/packages/devextreme-angular/src/common/charts.ts
?